### DISCUSSION TITLE
Collaboration Role Creation

### DISCUSSION THREAD

**Discussion by marcelomirandasilva** [Budibase/budibase#12582] [Help]
Is it possible to create a collaborator type role? With this role, the user could suggest changes to a given record, but to validate the operation, approval would be required from a user who had an Editor role.

**Comment by ConorWebb96** [comment]
Hey @marcelomirandasilva,

You could do something along these lines if you setup your RBAC correctly. Whatever role you decide, that would have this sort of functionality could have additional logic when interacting with any forms within your app. 

This could be something as simple as creating a new clone record in a separate table, which then has automations on it that would send an email to inform whoever needs informed that verification is needed. From here you could then pull this into another page. which would provide a means of saving over the original record if and when it is verified and accepted. If its not accepted it could be deleted and then a different automation can be created which would be sent explaining why it was declined.

Related docs to what i've discussed above can be found below;
- [Automations](https://docs.budibase.com/docs/automation-steps)
- [RBAC](https://docs.budibase.com/docs/in-app-user-management)
- [Email automation step](https://docs.budibase.com/docs/send-email)

I hope this makes sense.
